.tekton/odh-llm-d-inference-scheduler-v2-24-pull-request.yaml

```diff
@@ -39,7 +39,22 @@ spec:
     value: true
   - name: prefetch-input
     value: |
-      [{"type": "gomod", "path": "."}, {"path": ".", "type": "generic"}, {"type": "rpm", "path": "."}]
+      [
+        {"type": "gomod", "path": "."},
+        {"path": ".", "type": "generic"},
+        {
+          "type": "rpm",
+          "path": ".",
+          "include_summary_in_sbom": true,
+          "options": {
+            "dnf": {
+              "epel-9": {
```

Comment:
This identifier has to match the repo ID from the lockfile, otherwise this won't match anything.